File: examples/PokerData_2_Mapper.java
// Poker Data 2 Mapper

public class Mapper {
    public void map(String key, String value) {
        String data[] = key.split(",");
        String suit = data[0];
        String rank = data[10];
        String output = "";
        if(rank.equals("5") || rank.equals("8") || rank.equals("9")) {
            output = suit + ',' + rank;
            Wmr.emit(output, Integer.toString(1));
        }
    }
}

File: examples/poker/PokerData_2_Mapper.py
# Poker Data 2 Mapper

def mapper(key, value) :
    data = key.split(',')
    rank = data[10]
    if rank == "5" or rank == "8" or rank == "9" :
        suit = data[0]
        info = suit + "," + rank
        Wmr.emit(info, 1)